regolith/tools.py

```diff
@@ -1543,6 +1543,8 @@ def grant_burn(grant, appts, begin_date=None, end_date=None):
             pd_val += b.get('postdoc_months') * 30.5
         if b.get('ss_months'):
             ss_val += b.get('ss_months') * 30.5
+    if isinstance(appts, dict):
+        appts = collect_appts([{"appointments": appts}])
```

Comment:
@sbillinge  a little bit of cheating. if this is not very readable, I will added a parallel for-loop logic instead for the dict case, but this saves quite some space!